pydy/viz/scene.py

```diff
@@ -363,14 +363,10 @@ def generate_visualization_json_system(self, system, **kwargs):
                                           system.integrate(),
                                           system.constants.values(), **kwargs)
 
-    def create_static_html(self, overwrite=False, silent=False):
-
-        """Creates a directory named ``static`` in the current working
-        directory which contains all of the HTML, CSS, and Javascript files
-        required to run the visualization. Simply open ``static/index.html``
-        in a WebGL compliant browser to view and interact with the
-        visualization.
+    def create_static_html(self, copy=False, silent=False):
```

Comment:
In general we need to avoid changing call signatures of public functions. Any time this is done we have to deprecate the behavior.